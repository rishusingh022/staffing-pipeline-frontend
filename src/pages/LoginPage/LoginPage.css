.login-page{
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
    height: 100vh;
    background: #D9D9D9;
}
.card{
    display: flex;
    align-items: center;
    justify-content: center;
    flex-direction: column;
    gap:30px;
    width: 518px;
    height: 509px;
    background: #F8F9FA;
}
.card button{
    width: 289.71px !important;
    height: 36.48px !important;
    background-color:  #051B2C;
    font-family: 'Inter' !important;
    font-style: normal !important;
    font-weight: 400 !important;
    font-size: 15px !important;
    line-height: 18px !important;
    text-align: center !important;
}
.card h1{
    font-weight: 800;
    font-size: 15px;
    font-family: 'Inter';
}
