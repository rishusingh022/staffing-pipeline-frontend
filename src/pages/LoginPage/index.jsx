import * as React from 'react';
import './LoginPage.css';
import InputComponent from '../../components/InputComponent';
import Footer from '../../components/footer';
import Button from '../../components/Button';
import { Header } from '../../components';
const LoginPage = () => {
  const onSubmit = () => {
    console.log('onSubmit');
  };

  return (
    <div>
      <Header hasNav={false} />
      <div className="login-page">
        <div className={'card'}>
          <h1>Hub Capablities and staffing portal</h1>
          <InputComponent placeholder={'email'} />
          <InputComponent placeholder={'password'} />
          <Button buttonText={'login'} handleClick={onSubmit} />
        </div>
      </div>
      <Footer />
    </div>
  );
};

export default LoginPage;
