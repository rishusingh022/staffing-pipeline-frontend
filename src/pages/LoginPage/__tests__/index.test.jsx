import LoginPage from '..';
import { Fragment } from 'react';
import { render, screen } from '@testing-library/react';

describe('LoginPage Snapshot', () => {
  it('should render', () => {
    const { asFragment } = render(<LoginPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
