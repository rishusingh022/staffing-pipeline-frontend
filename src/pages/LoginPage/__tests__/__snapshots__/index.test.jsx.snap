// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage Snapshot should render 1`] = `
<DocumentFragment>
  <div>
    <div
      class="font-light h-[88px] text-white flex justify-center bg-deepBlue"
    >
      <div
        class="h-full xl:w-[63vw] lg:[70vw] md:[80vw] flex box-border"
      >
        <img
          class="h-[70px] self-end pl-0 lg:pr-[180px] sm:pr-[90px] pt-0 pb-[5px]"
          src="./assets/McK_Logo.png"
        />
      </div>
    </div>
    <div
      class="login-page"
    >
      <div
        class="card"
      >
        <h1>
          Hub Capablities and staffing portal
        </h1>
        <div>
          <input
            class="login-input"
            placeholder="email"
            type="text"
          />
        </div>
        <div>
          <input
            class="login-input"
            placeholder="password"
            type="text"
          />
        </div>
        <button
          class="button"
          data-testid="button"
        >
          login
        </button>
      </div>
    </div>
    <div
      class="footer"
    >
      <img
        alt="McKinsey and Company"
        src="mckinsey-logo.png"
      />
    </div>
  </div>
</DocumentFragment>
`;
